composer.json

```diff
@@ -11,7 +11,8 @@
     ],
     "require": {
         "roave/security-advisories": "dev-master",
-        "zendframework/zend-expressive": "^0.5"
+        "zendframework/zend-expressive": "^0.5",
+        "zendframework/zend-stdlib": "^2.7"
```

Comment:
Use 2.7, as we don't need hydrators. ;-)
On Oct 11, 2015 2:54 PM, "Marco Pivetta" <notifications@github.com> wrote:

> In composer.json
> <https://github.com/zendframework/zend-expressive-skeleton/pull/7#discussion_r41714643>
> :
>
> > @@ -11,7 +11,8 @@
> >      ],
> >      "require": {
> >          "roave/security-advisories": "dev-master",
> > -        "zendframework/zend-expressive": "^0.5"
> > +        "zendframework/zend-expressive": "^0.5",
> > +        "zendframework/zend-stdlib": "^2.7"
>
> Can rely on ~2.4 here, IMO.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendframework/zend-expressive-skeleton/pull/7/files#r41714643>
> .
>